normandy/selfrepair/static/js/normandy_driver.js

```diff
@@ -79,7 +129,22 @@ let Normandy = {
                 options.learnMoreMessage,
                 options.learnMoreUrl
             );
+
+            resolve(emitter);
```

Comment:
I'm imagining something like this:

1. `showHeartbeat` immediately fires an event (that is asynchronously dispatched)
2. `.then()` handler that would recieve the event emitter is registered (but not executed)

    > *Queue is now 1. handle event, 2. execute `.then()`*
4. Event is handled, and event emitter does nothing because there are no listeners

So this is really only an issue if `showHeartbeat` is going to immediately, synchronously, fire an event (which is reasonable). I don't know how it is implemented.
5. `.then()` handler runs, binding listener to the event emitter